test/unit/utils/sql.test.ts

```diff
@@ -264,3 +264,327 @@ describe('mapBindParameters', () => {
     });
   });
 });
+
+describe('injectReplacements (named replacements)', () => {
+  it('parses named replacements', () => {
+    const sql = injectReplacements(`SELECT ${dialect.TICK_CHAR_LEFT}:id${dialect.TICK_CHAR_RIGHT} FROM users WHERE id = ':id' OR id = :id OR id = ''':id'''`, dialect, {
+      id: 1,
+    });
+
+    expectsql(sql, {
+      default: `SELECT [:id] FROM users WHERE id = ':id' OR id = 1 OR id = ''':id'''`,
```

Comment:
Why are `dialect.TICK_CHAR_LEFT` and `dialect.TICK_CHAR_RIGHT` not needed in the result?

Reply:
`default` automatically converts `[` and `]` into the right tick for the dialect being tested (except in the case of `ARRAY[x]`)

Reply:
Ah, that makes sense. Thanks!